template/packages/typescript-config/vite.json

```diff
@@ -0,0 +1,8 @@
+{
+  "$schema": "https://json.schemastore.org/tsconfig",
+  "extends": "./base.json",
+  "compilerOptions": {
+    "lib": ["ESNext", "DOM"],
+    "useDefineForClassFields": true
```

Comment:
Why is this included here?

It looks like the default depends on your `target` which seems sensible at first glance.
https://www.typescriptlang.org/tsconfig/#useDefineForClassFields